src/sql/workbench/services/accountManagement/browser/accountDialog.ts

```diff
@@ -115,13 +116,6 @@ class AccountPanel extends ViewPane {
 	private updateTenants(account: azdata.Account) {
 		this.tenantList!.splice(0, this.tenantList!.length, account.properties?.tenants ?? []);
 	}
-
-	public getActions(): IAction[] {
-		return [this.instantiationService.createInstance(
-			AddAccountAction,
```

Comment:
Remove this old class if we're not using it anymore (although it looks like you have some references to it still, but those shouldn't be necessary anymore right?)

Reply:
Are you suggesting to remove `ViewPane` from `AccountPanel` entirely? I'm not quite sure if extending `ViewPane` was solely for the purpose of injecting actions...

Reply:
No, this is for removing the AddAccountAction (since that functionality is built into the registerAction2 calls you added)

Reply:
Oh that one, `registerAction2 ` will still eventually call `run()` from `AddAccountAction` with this line `v.addAccountAction.run();`